api-tests/data/cri-stub-requests/kbv/kenneth-score-0/evidence.json

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "IdentityCheck",
+  "verificationScore": 0,
```

Comment:
Is this realistic? Can we score 0 without it being a thin file case or raising a CI

Reply:
Yeah, for Experian KBVs both dropout and thin file are represented by a zero-scoring VC with no CI (unlike HMRC and DWP KBVs which conform to the new standard using `access_denied` and `invalid_request` oauth errors)